.github/workflows/test.yml

```diff
@@ -62,7 +63,7 @@ jobs:
         run: west zephyr-export
       - name: Test ${{ matrix.test }}
         working-directory: app
-        run: west test tests/${{ matrix.test }}
+        run: west test ${{ matrix.test }}
```

Comment:
We'll need to install the new Python deps before we run out tests here too. See the current CI failures.

Reply:
Yeah, I tried that, but the container doesn't have pip, so it will also fail if I try to install them.

Reply:
Yeah, we install pip to our build image then remove it again, to keep that image size small:

```
RUN \
  apt-get -y update \
  && if [ "$(uname -m)" = "x86_64" ]; then gcc_multilib="gcc-multilib"; else gcc_multilib=""; fi \
  && apt-get -y install --no-install-recommends \
  ccache \
  file \
  gcc \
  "${gcc_multilib}" \
  git \
  gperf \
  make \
  ninja-build \
  python3 \
  python3-dev \
  python3-pip \
  python3-setuptools \
  python3-wheel \
  && pip3 install \
  -r https://raw.githubusercontent.com/zephyrproject-rtos/zephyr/v${ZEPHYR_VERSION}/scripts/requirements-base.txt \
  && pip3 install cmake \
  && apt-get remove -y --purge \
  python3-dev \
  python3-pip \
  python3-setuptools \
  python3-wheel \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
```

We could either:

1. Install pip again, then install the deps, or
2. Add pytest to our Docker build images (I'm curious the change in image size, I can check this), or
3. Switch to the `dev` image just for this.

I'm tested to say #2 makes the most sense, if it doesn't bloat things a ton.

Reply:
Note that the tests also use the pythonsed package, not just pytest and pytest-xdist

Reply:
Ok, have a draft PR up: https://github.com/zmkfirmware/zmk-docker/pull/136 will inspect the images sizes to compare, the local builds w/ `buildah` here seemed just a *tiny* bit larger.